torchtext/experimental/datasets/text_classification.py

```diff
@@ -1,43 +1,13 @@
 import torch
 from torchtext.data.utils import get_tokenizer
-from torchtext.vocab import build_vocab_from_iterator
 from torchtext.experimental.datasets.raw import text_classification as raw
-
-
-def vocab_func(vocab):
-    def _forward(tok_iter):
-        return [vocab[tok] for tok in tok_iter]
-    return _forward
-
-
-def totensor(dtype):
-    def _forward(ids_list):
-        return torch.tensor(ids_list).to(dtype)
-    return _forward
-
-
-def ngrams_func(ngrams):
-    def _forward(token_list):
-        _token_list = []
-        for _i in range(ngrams + 1):
-            _token_list += zip(*[token_list[i:] for i in range(_i)])
-        return [' '.join(x) for x in _token_list]
-    return _forward
-
-
-def build_vocab(data, transforms):
```

Comment:
I'd leave `build_vocab` in the text classification file because the data is supposed to yield `label` and `txt`, which is not sure for other datasets.